website/templates/emails/pending_embargo.txt.mako

```diff
@@ -0,0 +1,20 @@
+Hello ${user.fullname},
```

Comment:
@bnosek @saradbowman This is the email template for embargo requests sent to project admins